src/org/labkey/test/tests/visualization/AssayRunFilterTest.java

```diff
@@ -0,0 +1,194 @@
+package org.labkey.test.tests.visualization;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.assay.GetProtocolCommand;
+import org.labkey.remoteapi.assay.ImportRunCommand;
+import org.labkey.remoteapi.assay.ImportRunResponse;
+import org.labkey.remoteapi.assay.Protocol;
+import org.labkey.remoteapi.assay.ProtocolResponse;
+import org.labkey.remoteapi.assay.SaveProtocolCommand;
+import org.labkey.remoteapi.domain.Domain;
+import org.labkey.remoteapi.domain.PropertyDescriptor;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.Charting;
+import org.labkey.test.categories.DailyC;
+import org.labkey.test.components.ChartTypeDialog;
+import org.labkey.test.pages.assay.AssayDataPage;
+import org.labkey.test.pages.assay.AssayRunsPage;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.PortalHelper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+@Category({DailyC.class, Charting.class})
+public class AssayRunFilterTest extends BaseWebDriverTest
```

Comment:
This test class seems really specific for testing this case. Did you consider adding this as another test case to an already existing Visualization test?

Reply:
I did.  I also considered factoring the assay setup into a helper method instead of building it all up.
I can move all of this stuff into scatterplotTest and scrap this one

Reply:
I think making this a test case in the `ScatterPlotTest` is a good idea. If you think this will take more than an hour though I'd say just leave it as-is.

Reply:
It is, and I looked at tacking this scenario into scatterplottest- but it was more pain to try to add a test to than this was to do.  This runs fast and works.

Reply:
OK, it's in there.
This eliminates an extra test class in that package, but it ties into a pattern I've been wanting to stamp out- specifically, the test where one method inherited via several base classes runs every test as a super-batch of aggregated tests that you can't run just one of.

Reply:
> I've been wanting to stamp out- specifically, the test where one method inherited via several base classes runs every test as a super-batch of aggregated tests that you can't run just one of.
👍 👍 👍  Not suggesting any changes to this PR but wanted to give my hearty support for this sentiment